modin/pandas/test/test_io.py

```diff
@@ -953,6 +993,21 @@ def test_read_csv_names_neq_num_cols(self, request, kwargs):
             **kwargs,
         )
 
+    def test_read_csv_wrong_path(self):
+
+        raising_exceptions = list(io_ops_bad_exc).remove(FileNotFoundError)
```

Comment:
this doesn't work as you expect, `.remove()` always returns `None` while simultaneously modifying its class.
```suggestion
        raising_exceptions = list(set(io_ops_bad_exc) - set([FileNotFoundError]))
```
or
```suggestion
        raising_exceptions = [e for e in io_ops_bad_exc if e != FileNotFoundError]
```